src/views/AddInstallation.vue

```diff
@@ -1,30 +1,593 @@
+        {{ $t("installation.addInRoom") }} {{ room.name }}
+      </h1>
+    </div>
+  </header>
+
+  <div v-if="isLoading">{{ $t("loading...") }}</div>
+  <div v-else>
+    <div v-if="isOwner">
+      <div v-if="hasSensors" class="max-w-sm sm:max-w-lg">
+        <div
+          class="
+            text-black
+            mt-2
+            p-4
+            card
+            rounded-md
+            shadow-md
+            ring-1 ring-gray-300
+            bg-white
+          "
+        >
+          <div class="form-control">
+            <label class="label">
+              <span class="label-text text-black font-bold">{{
+                $t("node.singular")
+              }}</span>
+            </label>
+            <Listbox v-model="selectedSensor">
+              <ListboxButton v-if="isSensorSelected"
+                >{{ selectedSensor?.alias }} (EUI:
+                {{ selectedSensor?.eui64 }})</ListboxButton
+              >
+              <ListboxButton v-else>{{
+                $t("installation.selectSensor")
+              }}</ListboxButton>
+              <ListboxOptions>
+                <ListboxOption
+                  v-for="sensor in sensors"
+                  :key="sensor._jv.id"
+                  :value="sensor"
+                >
+                  {{ sensor.alias }} (EUI: {{ sensor.eui64 }})
+                </ListboxOption>
+              </ListboxOptions>
+            </Listbox>
+          </div>
+          <div v-if="isSensorSelected">
+            <SwitchGroup>
+              <div class="flex items-center">
+                <SwitchLabel class="label-text text-black font-bold">{{
+                  $t("installation.makePublic")
+                }}</SwitchLabel>
+                <Switch
+                  v-model="isPublic"
+                  :class="isPublic ? 'bg-green' : 'bg-blue'"
+                  class="
+                    relative
+                    inline-flex
+                    items-center
+                    h-6
+                    rounded-full
+                    w-11
+                  "
+                >
+                  <span
+                    :class="isPublic ? 'translate-x-6' : 'translate-x-1'"
+                    class="
+                      inline-block
+                      w-4
+                      h-4
+                      transform
+                      bg-white
+                      shadow-lg
+                      rounded-full
+                      transition
+                    "
+                  />
+                </Switch>
+              </div>
+            </SwitchGroup>
+            <div class="form-control py-4">
+              <label class="label">
+                <span class="label-text text-black font-bold">{{
+                  $t("installation.from")
+                }}</span>
+              </label>
+              <Datepicker
+                v-model="fromDateTime"
+                locale="de"
+                weekNumbers
```

Comment:
The linter tell me to use hyphens for vue attributes. `weekNumbers` -> `week-numbers`. Same for the other attributes.

Reply:
which linter setup are you using?

Reply:
`eslint`. the config is in the repo `eslintrc.js`

Reply:
it's already installed via the dev dependencies. I'm using VSCode to edit

Reply:
To have VSCode show you the error without running the linter via the CLI (`yarn lint`) you can installed the ESLint extension in VSCode

Reply:
The linter isn't perfect. Some vue 3 specifics aren't identified correctly.

Reply:
fixed